### DISCUSSION TITLE
Is there any way I can submit a PR for the documentation?

### DISCUSSION THREAD

**Discussion by The-Briel-Deal** [pocketbase/pocketbase#408] [Q&A]
On https://pocketbase.io/docs/manage-collections/ it says "Collections are where you application data live.".
I was wondering if there is any way I can just fix this typo in the docs?

**Comment by ganigeorgiev** [comment]
_This was also discussed in https://github.com/pocketbase/pocketbase/discussions/36._

The documentation will have to be upated anyway because of https://github.com/pocketbase/pocketbase/issues/376 and I'll try to find time to reorganize the code and finally public the repo.

In the meantime, feel free to suggest your doc improvements by adding them here as comments/issues and I'll apply them manually.
